webui/master/src/containers/pages/Browse/Browse.tsx

```diff
@@ -215,7 +215,7 @@ export class Browse extends React.Component<AllProps, IBrowseState> {
                    `/browse?path=${path}${queryStringSuffix}`)}/>
           </FormGroup>
           <FormGroup className="mb-2 mr-sm-2">
-            <Button tag={Link} to={`/browse?path=${encodeURIComponent(path || "/")}${queryStringSuffix}`}
+            <Button tag={Link} to={`/browse?path=${encodeURIComponent(path || '/')}${queryStringSuffix}`}
```

Comment:
Unrelated to this PR, is it possible in typescript to infer that path cannot be undefined here?

Reply:
it depends how `path` is defined. In this case, it's defined as an optional string (`path?: string`) so typescript complains about that. I can look into simplifying this if possible